x-pack/plugins/security_solution/common/api/detection_engine/signals_migration/delete_signals_migration/delete_signals_migration.gen.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { z } from 'zod';
+
+/*
+ * NOTICE: Do not edit this file manually.
+ * This file is automatically generated by the OpenAPI Generator, @kbn/openapi-generator.
+ *
+ * info:
+ *   title: Alerts migration cleanup API endpoint
+ *   version: 2023-10-31
+ */
+
+export type CleanupAlertsMigrationRequestBody = z.infer<typeof CleanupAlertsMigrationRequestBody>;
+export const CleanupAlertsMigrationRequestBody = z.object({
+  migration_ids: z.array(z.string()).min(1).optional(),
+});
+
+export type CleanupAlertsMigrationResponse = z.infer<typeof CleanupAlertsMigrationResponse>;
+export const CleanupAlertsMigrationResponse = z.array(MigrationCleanupResult);
+
+export type MigrationCleanupResult = z.infer<typeof MigrationCleanupResult>;
+export const MigrationCleanupResult = z.object({
+  id: z.string(),
+  destinationIndex: z.string(),
+  status: z.enum(['success', 'failure', 'pending']),
+  sourceIndex: z.string(),
+  version: z.string(),
+  updated: z.string().datetime(),
+  error: z
+    .object({
+      message: z.string().optional(),
+      status_code: z.number().int().optional(),
```

Comment:
`message` and `status_code` within `error` are not optional, though `error` is optional